Tools/autotest/sim_vehicle.py

```diff
@@ -216,6 +216,13 @@ def find_root_dir():
     """Return path to root directory"""
     return os.path.realpath(os.path.join(find_autotest_dir(), '../..'))
 
+
+def wait_ulimited():
+    """Wait until Ctrl-C"""
+    time.sleep(2)
+    signal.pause()
```

Comment:
always forget about windows ... will look at another solution, maybe os.system("pause") on windows . The goal of this function is only to block the current term until sitl is closed, without it, when the new term is open, the script end.